### PR TITLE
Add mapping for Container entities

### PR DESCRIPTION
**Pull request by zakiva**
Adding a static mapping between openshift entities to miq entities.

### CODE REVIEW

File: app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb [hunk 1]
```diff
@@ -18,41 +18,49 @@ def get_openshift_images(inventory)
...
         end
       end
 
       def get_routes(inventory)
-        process_collection(inventory["route"], :container_routes) { |n| parse_route(n) }
+        process_collection(inventory["route"], map_entity("route")) { |n| parse_route(n) }
       end
 
       def get_projects(inventory)
+        projects = map_entity("project")
```
Comment (moolitayer) [inline app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb]: I don't know, maybe it's worth to:
```
data_index = map_entity("project")
```
?

That would bring us closer to extracting those methods (which we are not doing now)

**Comment by simon3z** [comment]
@cben @moolitayer can you review/approve?

**Comment by cben** [comment]
LGTM

**Comment by moolitayer** [comment]
@miq-bot assign @simon3z

**Pull request merged**